src/D2L.Bmx/Aws/AwsCacheModel.cs

```diff
@@ -0,0 +1,8 @@
+namespace D2L.Bmx.Aws;
+internal record AwsCacheModel(
+	string Org,
+	string User,
+	string AccountName,
+	string RoleArn,
```

Comment:
actually I wonder if it makes more sense to just save the role name here. the rest of the role arn isn't really used right?
no strong feeling though

Reply:
Sorry missed this one, we shouldn't need it because we have a account name in key, might as well just need to re-work the ``groupby`` expression a little